From 3caf545c358496f57952fd958d72215126660c5f Mon Sep 17 00:00:00 2001
From: alpha lin <alpha.lin@rock-chips.com>
Date: Fri, 1 Apr 2016 11:45:04 +0800
Subject: [PATCH] ARM64/cros_defconfig: remove IEP and RK_VCODEC

Remove CONFIG_IEP and CONFIG_RK_VCODEC definition for
they aren't required in rockchip chromeos.

Change-Id: I3a0bce0943931a7546378fb7c7e663e1317b93da
Signed-off-by: alpha lin <alpha.lin@rock-chips.com>
---
 arch/arm64/configs/rockchip_cros_defconfig | 2 --
 1 file changed, 2 deletions(-)

diff --git a/arch/arm64/configs/rockchip_cros_defconfig b/arch/arm64/configs/rockchip_cros_defconfig
index 42c66a061054..97be8a491a16 100644
--- a/arch/arm64/configs/rockchip_cros_defconfig
+++ b/arch/arm64/configs/rockchip_cros_defconfig
@@ -367,8 +367,6 @@ CONFIG_MIPI_DSI=y
 CONFIG_RK32_MIPI_DSI=y
 CONFIG_RK_HDMI=y
 CONFIG_ROCKCHIP_RGA2=y
-CONFIG_RK_VCODEC=y
-CONFIG_IEP=y
 CONFIG_LOGO=y
 # CONFIG_LOGO_LINUX_MONO is not set
 # CONFIG_LOGO_LINUX_VGA16 is not set
-- 
2.35.3

